api/STYLE.md

```diff
@@ -132,13 +134,15 @@ To add an extension config to the API, the steps below should be followed:
        )
    ```
 1. Add to the v2 extension config proto `import "udpa/annotations/migrate.proto";`
-2. Add to the v2 extension config proto a package level `option (udpa.annotations.file_migrate).move_to_package = "envoy.extensions.filters.http.foobar.v3";`.
-   This places the filter in the correct [v3 package hierarchy](#package-organization).
-3. Add a reference to the v2 extension config in (1) in [api/docs/BUILD](docs/BUILD).
-4. Run `./tools/proto_format fix`. This should regenerate the `BUILD` file,
+1. Add to the v2 extension config proto a file level `option (udpa.annotations.file_migrate).move_to_package = "envoy.extensions.filters.http.foobar.v3";`.
+   This places the filter in the correct [v3 package hierarchy](#package-organization). 
+1. If this is still WiP and subject to breaking changes, import
+   `udpa/annotations/status.proto` and set `option (udpa.annotations.file_status).work_in_progress = true;`.
+1. Add a reference to the v2 extension config in (1) in [api/docs/BUILD](docs/BUILD).
```

Comment:
There was confusion about this recently, but how are the v3 docs being built? Doesn't something have to be added for v3 also?

Reply:
#9719 added the mechanism: proto_format.sh runs tools generating api/BUILD, which contains `v2_protos` and `v3_protos` there, the source of proto_format.sh is `api/docs/BUILD` which only needs v2 to be added. So v3 references doesn't have to be added now.